generic/tests/cfg/kdump.cfg

```diff
@@ -18,14 +18,14 @@
         kdump_config = core_collector makedumpfile -c -d 31
     RHEL.6:
         kernel_param_cmd = "grubby --update-kernel=`grubby --default-kernel` --args=crashkernel=128M@16M"
-        kdump_enable_cmd = "chkconfig kdump on && service kdump start"
+        kdump_enable_cmd = "chkconfig kdump on && service kdump restart"
         kdump_propagate_cmd = "service kdump propagate"
         kdump_restart_cmd = "service kdump restart"
     RHEL.7:
         kernel_param_cmd = "grubby --update-kernel=`grubby --default-kernel` --args=crashkernel=auto"
-        kdump_enable_cmd = "chkconfig kdump on && kdumpctl start"
+        kdump_enable_cmd = "chkconfig kdump on && systemctl restart kdump.service"
```

Comment:
Thanks for your testing and will make it longer as this is not our key point for our testing.